modules/apps/commerce/commerce-payment-service/service.xml

```diff
@@ -29,11 +29,14 @@
 		<column name="currencyCode" type="String" />
 		<column name="errorMessages" type="String" />
 		<column name="languageId" type="String" />
+		<column name="note" type="String" />
 		<column name="paymentIntegrationKey" type="String" />
 		<column name="paymentIntegrationType" type="int" />
 		<column name="paymentStatus" type="int" />
+		<column name="reason" type="String" />
```

Comment:
shouldn't we have reason key and reason name?
So that we are protected against reason configuration changes, but still we can change the reason in the refund